syntax.d/groovy.ini

```diff
@@ -0,0 +1,9 @@
+name=Groovy
+extensions=groovy
+highlight_numbers=true
+singleline_string_quotes="
+singleline_comment_start=//
+multiline_comment_delims=/*, */
+multiline_string_delim="
```

Comment:
I'm not familiar with Groovy, but it looks like it uses """ for multi-line string literals? https://www.aurorasolutions.io/post/groovy-multi-line-strings

Reply:
Ah right https://groovy-lang.org/syntax.html#_string_summary_table